lib/lightstep/span.rb

```diff
@@ -5,9 +5,12 @@ module LightStep
   #
   # See http://www.opentracing.io for more information.
   class Span
+    # Part of the OpenTracing API
+    attr_writer :operation_name
+
     # Internal use only
     # @private
-    attr_reader :guid, :trace_guid, :start_micros, :end_micros, :baggage, :tags
+    attr_reader :guid, :trace_guid, :start_micros, :end_micros, :baggage, :tags, :operation_name
```

Comment:
if you're accessing it outside of the span class (in this case, you're accessing it from the test) then it can't be ruby private.